src/test/scala/com/gu/apis/SlackApiChannels.scala

```diff
@@ -43,8 +41,12 @@ case class SlackApiChannels(channel: String) {
     }
   }
 
+  def jsonToString(js: JsValue): String = js.toString.replace("\"", "")
+
   def getJsonAttribute(js: JsValue, attribute: String): String = {
-    jsonToString(js \ attribute)
+    jsonToString((js \ attribute).get)
```

Comment:
Given these helpers can't be consistently applied, would it be simpler to inline this one and create a helper to do the actual logic you care about (the replacement)?

```scala
def aNameAlongTheLinesOfExtractStringAndRemoveQuotes(js: JsValue): String = {
  js.as[String]._filterNot('"')
}
```

Then you can call it consistently (or not) as such:
```scala
goodName(latestMessageFirstAttachment \ "text")
goodName((latestMessageFirstAttachment \ "fields")(0))
(firstAttachmentFirstField \ "short").as[Boolean]
```

Names are hard, but maybe `jsonToString` is right for this new one still?